app/src/androidTest/java/com/github/polypoly/app/map/LocationRepositoryTest.kt

```diff
@@ -3,15 +3,15 @@ package com.github.polypoly.app.map
 import org.junit.Test
 
 // These tests are not really useful, but they're here symbolically, this is only a data class
-class LocalizationRepositoryTest {
+class LocationRepositoryTest {
     @Test
     fun testGetLocalizationsWorks() {
-        assert(LocalizationRepository.getZones().flatMap { zone -> zone.localizations }
+        assert(LocationRepository.getZones().flatMap { zone -> zone.locations }
             .isNotEmpty())
```

Comment:
This can be refactored into `assert(LocationRepository.getZones().flatMap(Zone::locations).isNotEmpty())` (usually `{x -> x.xyz}` can be replaced by `(X::xyz)` to make it more concise and more expressive, where `xyz` is a an attribute or a member function of the class `X`).

Reply:
I sure wish I could as a Scala enjoyer, but I found a neater workaround, hope it's good enough:
assert(LocationRepository.getZones().flatMap { it.locations }.isNotEmpty())

Reply:
Fellow Scala enjoyer, this should be doable in kotlin too, the code I sent above `assert(LocationRepository.getZones().flatMap(Zone::locations).isNotEmpty()) (usually {x -> x.xyz}` works on my machine 😄

Reply:
Indeed seems to work, IDE probably needed a good reboot

Reply:
Great 👍